redash/query_runner/vertica.py

```diff
@@ -126,16 +126,12 @@ def run_query(self, query, user):
             logger.debug("Vertica running query: %s", query)
             cursor.execute(query)
 
-            # TODO - very similar to pg.py
             if cursor.description is not None:
-                columns_data = [(i[0], i[1]) for i in cursor.description]
-                
-                # bug -- same name columns will always be assigned single last value
-                # corrected with fetch_columns in pull request https://github.com/getredash/redash/pull/4201
-                rows = [dict(zip((c[0] for c in columns_data), row)) for row in cursor.fetchall()]
-                columns = [{'name': col[0],
-                            'friendly_name': col[0],
-                            'type': types_map.get(col[1], None)} for col in columns_data]
+                columns_data = [(i[0], types_map.get(i[1], None)) for i in cursor.description]
+
+                columns = self.fetch_columns(column_data)
```

Comment:
Hi, I stumbled upon this commit while rebasing my work on master.
Tell me if I'm missing something but it looks like `column_data` is not defined anywhere. Shouldn't it be `columns_data`?

Reply:
:flushed: of course you're right.

Reply:
#4210